core/src/geom/utils.cpp

```diff
@@ -191,3 +192,16 @@ void cubos::core::geom::getCameraFrustumCorners(const glm::mat4& view, const glm
         }
     }
 }
+
+void cubos::core::geom::getCubeViewMatrices(const glm::mat4& inverseView, std::vector<glm::mat4>& cubeViewMatrices)
+{
+    static const float ViewAngle[6] = {0.0F, 180.0F, 90.0F, -90.0F, 90.0F, -90.0F};
+    static const glm::vec3 ViewAxis[6] = {glm::vec3(0.0F, 1.0F, 0.0F), glm::vec3(0.0F, 1.0F, 0.0F),
+                                          glm::vec3(0.0F, 1.0F, 0.0F), glm::vec3(0.0F, 1.0F, 0.0F),
+                                          glm::vec3(1.0F, 0.0F, 0.0F), glm::vec3(1.0F, 0.0, 0.0F)};
+
+    for (int i = 0; i < 6; i++)
+    {
+        cubeViewMatrices.push_back(glm::inverse(glm::rotate(inverseView, glm::radians(ViewAngle[i]), ViewAxis[i])));
+    }
```

Comment:
```suggestion
    
    cubeViewMatrices.resize(6);
    for (int i = 0; i < 6; i++)
    {
        cubeViewMatrices[i] = glm::inverse(glm::rotate(inverseView, glm::radians(ViewAngle[i]), ViewAxis[i]));
    }
```
Just to avoid some memory allocations :relieved: